neware_parser/PlotEngine.py

```diff
@@ -0,0 +1,120 @@
+import pickle
+
+import matplotlib.patches as mpatches
+
+# TODO(harvey) duplicate in plot.py
+COLORS = [
+    (.4, .4, .4),
+
+    (1., 0., 0.),
+    (0., 0., 1.),
+    (0., 1., 0.),
+
+    (.6, 0., .6),
+    (0., .6, .6),
+    (.6, .6, 0.),
+
+    (1., 0., .5),
+    (.5, 0., 1.),
+    (0., 1., .5),
+    (0., .5, 1.),
+    (1., .5, 0.),
+    (.5, 1., 0.),
+]
+
+
+class PlotEngine:
+
+    @staticmethod
```

Comment:
storing a single scalar in a pickle file is dumb.
you probably want to store all the data from a single barcode in the same pickle file.

Your loader should be trivial (i.e. you just load a big dictionary that you pickled)

Then, you have a function you can call with this big structure that knows how to plot 
![voltage_dependence_57715](https://user-images.githubusercontent.com/30840933/80770495-d95d2600-8b26-11ea-8577-9010a0545aa2.png)
or 
![Cap_82992_Count_10000](https://user-images.githubusercontent.com/30840933/80770548-f8f44e80-8b26-11ea-9792-8f5515be116b.png)
or 
![VQ_82992_Count_100000](https://user-images.githubusercontent.com/30840933/80770581-09a4c480-8b27-11ea-9162-dc6e4cf3de38.png)
and you do that by calling smaller functions that can for instance plot_scale, or something else.
Then you call another function to save the figure to a file.